CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Added
 
 - Added support for UTF-8 filenames.
-- Added `example-component-diagram.tex`. Refs [#2](https://github.com/koppor/plantuml/issues/9).
```

Comment:
Do you want to take over maintenance of this package?

Reply:
No, it's mis-operation. The original PR (pushed more than 8 days ago) didn't get merged for a long time, I can't wait to use it so I changed something try to publish my own one to use.

But finally I found I can just use it by put a local `plantuml.sty` so it is OK not to wait for new packages.

Will close it, sorry.